virtual_ecosystem/core/registry.py

```diff
@@ -184,6 +192,46 @@ def register_module(module_name: str) -> None:
                 class_name,
             )
 
+    # Find and register the model configuration
+    try:
+        config_submodule = import_module(f"{module_name}.model_config")
+    except ModuleNotFoundError:
+        raise RuntimeError("Model does not provide a model_config submodule.")
```

Comment:
Maybe indicate the offending model in the message:

```suggestion
        raise RuntimeError(f"Model `{module_name}` does not provide a model_config submodule.")
```